rdt/transformers/text.py

```diff
@@ -1,251 +1,12 @@
-                    self.generator_size,
-                    self.get_input_column(),
-                )
-
-        remaining = self.generator_size - self.generated
-        if sample_size > remaining and self.enforce_uniqueness and not warned:
-            warnings.warn(
-                f'The regex generator is not able to generate {sample_size} new unique '
-                f'values (only {max(remaining, 0)} unique values left).'
-            )
-
-    def _reverse_transform(self, data):
-        """Generate new data using the provided ``regex_format``.
-
-        Args:
-            data (pd.Series or numpy.ndarray):
-                Data to transform.
-
-        Returns:
-            pandas.Series
-        """
-        if data is not None and len(data):
-            sample_size = len(data)
-        else:
-            sample_size = self.data_length
-
-        self._warn_not_enough_unique_values(sample_size)
-
-        remaining = self.generator_size - self.generated
-        if sample_size > remaining:
-            self.reset_randomization()
-            remaining = self.generator_size
-
-        generated_values = []
-        while len(generated_values) < sample_size:
-            try:
-                generated_values.append(next(self.generator))
-                self.generated += 1
-            except (RuntimeError, StopIteration):
-                # Can't generate more rows without collision so breaking out of loop
-                break
-
-        reverse_transformed = generated_values[:]
-
-        if len(reverse_transformed) < sample_size:
-            if self.enforce_uniqueness:
-                try:
-                    remaining_samples = sample_size - len(reverse_transformed)
-                    start = int(generated_values[-1]) + 1
-                    reverse_transformed.extend([
-                        str(i) for i in range(start, start + remaining_samples)
-                    ])
-
-                except ValueError:
-                    counter = 0
-                    while len(reverse_transformed) < sample_size:
-                        remaining_samples = sample_size - len(reverse_transformed)
-                        reverse_transformed.extend([
-                            f'{i}({counter})' for i in generated_values[:remaining_samples]
-                        ])
-                        counter += 1
-
-            else:
-                while len(reverse_transformed) < sample_size:
-                    remaining_samples = sample_size - len(reverse_transformed)
-                    reverse_transformed.extend(generated_values[:remaining_samples])
-
-        if getattr(self, 'generation_order', 'alphanumeric') == 'scrambled':
-            np.random.shuffle(reverse_transformed)
-
-        return np.array(reverse_transformed, dtype=object)
+warnings.warn(
+    "Importing 'IDGenerator' or 'RegexGenerator' for ID columns from 'rdt.transformers.text' "
+    "is deprecated. Please use 'rdt.transformers.id' instead.",
```

Comment:
This warning mentions importing from `transformers.id` instead of `transformers.text` but should it also mention switching the sdtype from text to id?

Reply:
Not sure, cc: @npatki

Reply:
Addressed here [2a8d014](https://github.com/sdv-dev/RDT/pull/881/commits/2a8d014d84d1422b66fc8fe6b3d97502c67220b0)